# Exception handling
try:
    
    b=[1,2,3,2,1,2,]
    print(b[10])
except Exception as e:
    print(e)
print("Number is not divide by Zero")